swift/aperture/Recorder.swift

```diff
@@ -1,13 +1,22 @@
 import AVFoundation
 
+enum ApertureError: Error {
+  case couldNotAddScreen
+  case couldNotAddMic
+  case couldNotAddOutput
```

Comment:
I dont think there are any established conventions yet. I think this is the most used too.